src/coreclr/src/System.Private.CoreLib/ILLinker/ILLink.Suppressions.cs

```diff
@@ -0,0 +1,276 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: UnconditionalSuppressMessage ("ILLinker", "IL2006", Scope = "member", Target = "F:System.LazyDebugView`1._lazy")]
```

Comment:
I'd make sense to have the warnings for shared SPC in the shared location (it could turn out there are not any in runtime specific code)